docs/Earn/bridge/bridge-mechanics.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Ankr Bridge Mechahics
+id: bridge-mechanics
+---
+
+# Ankr Bridge mechahics
+
+# Briding workflow
+
+The mechanics of what's going on under the hood when a user transfers their funds between network can be described as following:
+
+1. The user has Ankr Liquid Staking tokens on the network where our Staking smart contract minted them for said user.
+
+2. The users wants to transfer them on another network. For that they visit [Ankr Bridge](https://www.ankr.com/earn/bridge/), choose the amount to be transferred and click **Approve**. 
+   That causes Ankr Bridge to call the `approve()` function of the briding smart contract to allow transferring the assets. From the user's side, it looks like they approve the operation in their Metamask.  
```

Comment:
bridging
to allow transfer of the assets.
From the user side, this function appears as an 'Approve' message....

Reply:
thx